calkit/templates/core.py

```diff
@@ -1,5 +1,7 @@
 """Core functionality for working with templates."""
 
+from __future__ import annotations
```

Comment:
This may be related to one of the issues in https://github.com/calkit/calkit-assistant/issues/24